tests/functional/modules/test_zos_archive_func.py

```diff
@@ -362,6 +363,40 @@ def test_uss_archive_encode(ansible_zos_module, ds_format):
         hosts.all.file(path=USS_TEMP_DIR, state="directory")
         set_uss_test_env(hosts, USS_TEST_FILES)
         dest = f"{USS_TEMP_DIR}/archive.{ds_format}"
+        archive_result = hosts.all.zos_archive(
+            src=list(USS_TEST_FILES.keys()),
+            dest=dest,
+            format={
+                "name":ds_format
+            },
+            encoding={
+                "from": TO_ENCODING,
```

Comment:
This confuses me: encoding/from is to_encoding, and encoding/to is from_encoding.  Should it be the other way around?  If this is correct, I'd suggest adding a comment or two to explain the switch.